R/Simulations-class.R

```diff
@@ -388,44 +388,38 @@ DualSimulations <- function(rho_est,
   ))
 }
 
-# nolint start
 # DualSimulationsSummary ----
 
-##' Class for the summary of dual-endpoint simulations output
-##'
-##' In addition to the slots in the parent class
-##' \code{\linkS4class{SimulationsSummary}}, it contains two slots for the
-##' biomarker model fit information.
-##'
-##' Note that objects should not be created by users, therefore no
-##' initialization function is provided for this class.
-##'
-##' @slot biomarkerFitAtDoseMostSelected fitted biomarker level at dose most often selected
-##' @slot meanBiomarkerFit list with the average, lower (2.5%) and upper (97.5%)
-##' quantiles of the mean fitted biomarker level at each dose level
-##'
-##' @export
-##' @keywords classes
+# class ----
+
+#' `DualSimulationsSummary`
+#'
+#' @description `r lifecycle::badge("stable")`
+#' This class captures the summary of dual-endpoint simulations output.
+#' In comparison to parent class [`SimulationsSummary`], it has additional slots.
```

Comment:
```suggestion
#' In comparison to its parent class [`SimulationsSummary`], it has additional slots.
```

Reply:
I have adjusted a few methods like plot and summary in R/Simulations-methods.R and tested to see there is no error anymore in the example.Rmd and few other vignettes. Do you think there are other places I should look at?